crates/analyze/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "contract-analyze"
 version = "4.1.1"
-authors = ["Parity Technologies <admin@parity.io>"]
+authors = ["Use Ink <ink@r0gue.io>"]
```

Comment:
Can we set something up to use use.ink instead?

Reply:
That's a good point. Is it possible for you set up `ink@use.ink` once the domains have been transferred?

Reply:
Yeah, we can set it up as the existing one used. Please do let us know if you have any specific address in mind?

Would suggest that all other services currently being set up are then transitioned to this new address, but this can be done behind the scenes after the initial transition.